src/components/ha-card.ts

```diff
@@ -14,26 +14,26 @@ export class HaCard extends LitElement {
           --ha-card-background,
           var(--card-background-color, white)
         );
-        border-radius: var(--ha-card-border-radius, 4px);
-        box-shadow: var(
```

Comment:
removing this makes that it is no longer themeable and thus a breaking change, is that intended?

Reply:
It's in the raised option. If we let this by default, all themes that override `ha-card-box-shadow` will have border + box shadow. 
I don't known what is the better option